automatic/apache-httpd/apache-httpd.nuspec

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <metadata>
+    <id>apache-httpd</id>
+    <version>2.4.25</version>
+    <title>Apache HTTP Server Project</title>
+     <authors>Apache Software Foundation</authors>
+    <owners>chocolatey</owners>
+    <licenseUrl>http://www.apache.org/licenses</licenseUrl>
+    <projectUrl>http://httpd.apache.org</projectUrl>
+    <docsUrl>http://httpd.apache.org/docs/2.4</docsUrl>
+    <iconUrl>https://cdn.rawgit.com/chocolatey/chocolatey-coreteampackages/2bdf6f7e33ec1a8126829fbbc87b83e4473b3634/icons/apache-httpd.png</iconUrl>
+    <bugTrackerUrl>http://httpd.apache.org/bug_report.html</bugTrackerUrl>
+    <releaseNotes>http://httpd.apache.org/docs/2.4/new_features_2_4.html</releaseNotes >
+    <packageSourceUrl>https://github.com/chocolatey/chocolatey-coreteampackages/tree/master/automatic/apache-httpd</packageSourceUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <summary>The Apache HTTP Server Project is a collaborative software development effort aimed at creating a robust, commercial-grade, featureful, and freely-available source code implementation of an HTTP (Web) server.</summary>
+    <description>
+    </description>
+    <tags>apache httpd webserver admin</tags>
+    <dependencies>
+       <dependency id="vcredist2015" version="14.0" />
+       <dependency id="chocolatey-core.extension" version="1.0.5" />
```

Comment:
@majkinetor Isn't the correct way to specify the minimum for chocolatey-core.extension?